myapp/config/locales/ja.yml

```
@@ -213,4 +213,11 @@ ja:
         edited: 'タスクを編集しました'
         failed_to_edited: 'タスクを編集に失敗しました'
         deleted: 'タスクを削除しました'
+  enums:
+    task:
+      status:
+        yet: '未着手'
+        wip: '着手中'
```

個人的には`yet`は`なお`とかの意味もあるのと`wip`は文字数的には気持ちがいいですが略語なので↓とかがいいかなと思いました。
```suggestion
        open: '未着手'
        doing: '着手中'
```
あくまで個人的意見なので参考までに！

---

>あくまで個人的意見なので参考までに！

私は命名のセンスが無いので、助かります。
こちらの方がよりリーダブルだと思いました。
修正します。